app.json

```diff
@@ -490,6 +490,22 @@
       "description": "The api url for xpro courses",
       "required": false
     },
+    "XPRO_ITERATOR_CHUNK_SIZE": {
+      "description": "Chunk size for iterating over xPRO courses for master json",
+      "required": false
+    },
+    "XPRO_LEARNING_COURSE_BUCKET_NAME": {
+      "description": "Name of S3 bucket to upload xPRO course media",
+      "required": false
+    },
+    "XPRO_LEARNING_COURSE_ACCESS_KEY": {
```

Comment:
Maybe XPRO_LEARNING_COURSE_ACCESS_KEY_ID to match the amazon documentation?

Reply:
That makes sense but I'd like to keep it the way it is so that it matches with `OCW_LEARNING_COURSE_ACCESS_KEY`